roles/tpa_single_node/templates/manifests/vexination/walker/Deployment.yaml.j2

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vexination-walker
+  namespace: "{{ tpa_single_node_namespace }}"
+  labels:
+    name: vexination-walker
+    app: vexination-walker
+spec:
+  selector:
+    matchLabels:
+      app: vexination-walker
+  template:
+    spec:
+      restartPolicy: Never
+      volumes:
+        - name: walker-state
+          persistentVolumeClaim:
+            claimName: vexination-walker-state
+      containers:
+        - image: "{{ tpa_single_node_trustification_image }}"
+          imagePullPolicy: Always
+          name: walker
+          command:
+            - /trust
+          args:
+            - vexination
+            - walker
+            - --sink
+            - http://vexination-api-pod:8080/api/v1/vex
```

Comment:
I believe this needs to be replaced with 127.0.0.1 for the host and 8081 for the port. That ports need to be set-up in vexination api deployment manifest as we cannot use 8080.